typescript/sdk/src/token/config.ts

```diff
@@ -12,6 +12,7 @@ export enum TokenType {
   fastCollateral = 'fastCollateral',
   collateralUri = 'collateralUri',
   native = 'native',
+  // backwards compatible alias to native
```

Comment:
Should we up the sdk major version and just drop this instead?

Reply:
🤷 I figured it was nice that this didnt need to break SDK and CLI users with existing config

Reply:
agree it's nice not to break existing config users, but we'll end up with a major SDK release for this anyway right because of the major changeset in its `core` dependency?

Reply:
I'm fine either way, up to you guys

Reply:
https://github.com/changesets/changesets/blob/main/docs/fixed-packages.md we decoupled the core package from the rest of the typescript packages as an action item of https://www.notion.so/hyperlanexyz/Contracts-Release-Process-6b516e90b8db4e519a9f4881d53f7822?pvs=4